src/java/bobbybot/BobbyBot.java

```diff
@@ -0,0 +1,60 @@
+package bobbybot;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class BobbyBot {
+    private final String DBPATH = "data/database.txt";
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    /**
+     * Constructor to initialise classes
+     */
+    public BobbyBot()  {
```

Comment:
There seems to be an extra white space here?

Reply:
Yep! Thanks for the comment. Will fix it!